crates/miden-objects/src/account/code/procedure.rs

```diff
@@ -149,6 +152,64 @@ impl Deserializable for AccountProcedureInfo {
     }
 }
 
+// PRINTABLE PROCEDURE
+
+/// A printable representation of a single account procedure.
+#[derive(Debug, Clone)]
+pub struct PrintableProcedure {
+    mast: Arc<MastForest>,
+    procedure_info: AccountProcedureInfo,
+    node_raw: MastNode,
```

Comment:
is `node_raw` the root node of the procedure? If so, I'd call it something like `root` or `entrypoint`.